src/Ionide.ProjInfo/Library.fs

```
@@ -792,60 +815,46 @@ type WorkspaceLoaderViaProjectGraph private (toolsPath, ?globalProperties: (stri
-                    |> Seq.map (fun p -> p.FullPath, ProjectLoader.getLoadedProjectInfo p.FullPath customProperties (ProjectLoader.LoadedProject p))
-
-                    |> Seq.choose (fun (projectPath, projectOptionResult) ->
-                        match projectOptionResult with
-                        | Ok projectOptions ->
-
-                            Some projectOptions
-                        | Error e ->
-                            logger.error (Log.setMessage "Failed loading projects {error}" >> Log.addContextDestructured "error" e)
-                            loadingNotification.Trigger(WorkspaceProjectState.Failed(projectPath, GenericError(projectPath, e)))
-                            None)
-
-                let allProjectOptions = projects |> Seq.toList
-
-                allProjectOptions
-                |> Seq.iter (fun po ->
-                    logger.info (Log.setMessage "Project loaded {project}" >> Log.addContextDestructured "project" po.ProjectFileName)
-                    loadingNotification.Trigger(WorkspaceProjectState.Loaded(po, allProjectOptions |> Seq.toList, false)))
-
-                allProjectOptions :> seq<_>
-        with
-        | e ->
-            let msg = e.Message
-
-            logger.error (Log.setMessage "Failed loading" >> Log.addExn e)
-
-            projects.ProjectNodesTopologicallySorted
-            |> Seq.distinctBy (fun p -> p.ProjectInstance.FullPath)
-            |> Seq.iter (fun p ->
-
-                let p = p.ProjectInstance.FullPath
-
-                if msg.Contains "The project file could not be loaded." then
-                    loadingNotification.Trigger(WorkspaceProjectState.Failed(p, ProjectNotFound(p)))
-                elif msg.Contains "not restored" then
-                    loadingNotification.Trigger(WorkspaceProjectState.Failed(p, ProjectNotRestored(p)))
+                if result.Exception |> isNull |> not then
+                    handleError result.Exception
```

Changed this because we would be hiding this exception if the build failed.